pkg/agent/templates/templates_windows.go

```diff
@@ -165,6 +168,20 @@ func ParseTemplateFromConfig(templateBuffer string, config interface{}) (string,
 		"replace": func(s string) string {
 			return strings.ReplaceAll(s, "\\", "\\\\")
 		},
+		"deschemify": func(s string) string {
```

Comment:
Drop the `logrus.Infof` calls out of this function; they appear to be mostly for debugging and won't mean anything to a user.